src/RichText/RichText.php

```diff
@@ -351,11 +351,22 @@ private static function fixImagesPath(string $content): string
         ];
 
         foreach ($patterns as $pattern) {
-            $content = preg_replace(
+            $result = preg_replace(
                 $pattern,
                 sprintf(' $1="%s/front/document.send.php$2" ', $CFG_GLPI["root_doc"]),
                 $content
             );
+            if ($result === null) {
+                $log_msg =
+                    "preg_replace failed in " . __METHOD__ .
+                    " with pattern: $pattern\n" .
+                    "error: " . preg_last_error_msg() . "\n" .
+                    "content length: " . strlen($content) . "\n" .
+                    "content start: " . substr($content, 0, 200);
```

Comment:
Logging content could have some security concerns (if the log file leak, some ticket content could be exposed).